### PR TITLE
Use new StatisticsService to send stats to collector

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

**Comment by anthonydahanne** [comment]
needs rebasing

**Comment by henri-tremblay** [comment]
Rebased

File: core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ */
+
+package org.ehcache.core.statistics;
+
+import java.util.Map;
+
+/**
+ * All statistics relative to a cache and its underlying tiers.
+ *
+ * @author Henri Tremblay
```
Comment (ljacomet) [inline core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java]: Please configure your IDE to not generate these, the source control systems does a much better job

**Comment by mathieucarbou** [comment]
@henri-tremblay : needs to be rebased ;-)

File: core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java (hunk 1 above)
Comment (henri-tremblay) [inline core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java]: Ok. Tags removed

**Comment by anthonydahanne** [comment]
test this please
infrastructure issue (git checkout problem)

**Pull request merged**